tests/main/views/test_buyers.py

```diff
@@ -1689,7 +1737,9 @@ def test_show_live_brief_summary_page_for_live_and_expired_framework(self, data_
 
                 assert not document.xpath('//a[contains(text(), "Delete")]')
 
-    def test_show_closed_brief_summary_page_for_live_and_expired_framework(self, data_api_client):
+    @pytest.mark.parametrize('status', ['closed', 'cancelled', 'unsuccessful'])
```

Comment:
Typo in `canclled`

Reply:
Also fixed in the other PR: https://github.com/alphagov/digitalmarketplace-briefs-frontend/pull/21/files#diff-9a0721b5c761c2fccb366192a3212493R1785

Basically in the other PR we have reorganised this class into one test for each status, except for cancelled/unsuccessful which are almost identical.